appveyor.yml

```diff
@@ -11,8 +11,12 @@ environment:
     #   BUNDLE_JOBS: 4
     - RUBY_VERSION: 24-x64
       USE_MSYS: true
-      SUITES: "spec acceptance:local"
+      SUITES: "acceptance:local"
       BUNDLE_JOBS: 4
+    - RUBY_VERSION: 24-x64
+      USE_CYGWIN: true
```

Comment:
To keep the same environment and coverage as before, shouldn't this line be `USE_MSYS: true` ?

Reply:
:+1: fixed